test-framework/maven/src/main/java/io/quarkus/maven/it/verifier/MavenProcessInvoker.java

```diff
@@ -48,7 +48,7 @@ public InvocationResult execute(InvocationRequest request) throws MavenInvocatio
 
         File workingDirectory = getWorkingDirectory();
         if (workingDirectory != null) {
-            cliBuilder.setWorkingDirectory(getWorkingDirectory());
+            cliBuilder.setBaseDirectory(getWorkingDirectory());
```

Comment:
See https://github.com/apache/maven-invoker/commit/ca84e5c6108366b800c2e61d377eadcb327d6499#diff-7a2ee223cf0ee38d98ed325df11438abe1b5134ce5d7430933461efc0ff5963cL725
I think this change should be ok given the way Quarkus is using this method.

Reply:
This is a change in `maven-invoker`, not in actual Maven, so I don't think Maven < 3.8.6 will have a problem with this.

Should I break this up in three PRs?